src/convert.jl

```diff
@@ -1225,3 +1225,25 @@ function convert(::Type{Hyperplane}, P::HPolyhedron; skip_check::Bool=false)
     c1 = @inbounds first(constraints_list(P))
     return Hyperplane(c1.a, c1.b)
 end
+
+
+"""
+    convert(::Type{SimpleSparsePolynomialZonotope}, P::Zonotope)
+
+Convert a zonotope to a simple sparse polynomial zonotope.
+
+### Algorithm
+
+This method implements Proposition 3 in [1].
+
+[1] Kochdumper, Althoff. *Sparse polynomial zonotopes - a novel set
+representation for reachability analysis*. 2021
+"""
+function convert(::Type{SimpleSparsePolynomialZonotope}, P::Zonotope)
```

Comment:
Let's dispatch on `P::AbstractZonotope` since they define `center`, `genmat` and `ngens` (should do if not). This way we can reuse this method for `Hyperrectangle` and other zonotopic sets.

Reply:
I would also rename `P` to `Z`.